lsp-completion.el

```diff
@@ -37,6 +37,7 @@
   "The completion backend provider."
   :type '(choice
           (const :tag "Use company-capf" :capf)
+          (const :tag "Use simplified capf" :corfu-capf)
           (const :tag "None" :none))
```

Comment:
Maybe this option should be replaced with a new option `lsp-completion-frontend` which can take these values:

- `default-passthrough`: Setup passthrough style, use label
- `default-filtered`: Setup flex style (or orderless if available), use filter-text
- `company`: Auto configure Company, like `default-passthrough`
- `corfu`: Auto configure Corfu, like `default-filtered`